### PR TITLE
Added recipe for `pyrobex`

### PR DESCRIPTION
**Pull request by sarthakpati**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by sarthakpati** [comment]
@conda-forge/help-python this is ready for merge!

File: recipes/pyrobex/LICENSE [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (dopplershift) [inline recipes/pyrobex/LICENSE:1]: This is already included in the PyPI tarball so you shouldn't include it manually in the recipe.

File: recipes/pyrobex/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 99b51c3139dd0f9402d69fc1041882769f3d1bea210a9e6cebf25470c4f53632
+
+build:
+  number: 0
+  noarch: python
+  skip: True
```
Comment (dopplershift) [inline recipes/pyrobex/meta.yaml:16]: Need to remove this since you're effectively skipping all builds.
```suggestion
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Let's see if it works after this change.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**